core/src/main/java/org/infinispan/cache/impl/CacheImpl.java

```diff
@@ -1112,7 +1176,7 @@ final V put(K key, V value, Metadata metadata, long explicitFlags) {
 
    @SuppressWarnings("unchecked")
    private V putInternal(K key, V value, Metadata metadata,
-         long explicitFlags, InvocationContext ctx) {
+                         long explicitFlags, InvocationContext ctx) {
```

Comment:
You've caused a lot of reformattings. It would be better to have whitespace-only ones in separate commit/PR - though I am not asking to do it here by hand (isolating such changes via script would be ultracool!).